README.md

```diff
@@ -36,7 +36,7 @@ A tool that I thought we also ship that would easily verify your “empty” PDB
 
 So in summary, by using the externally defined function entry points in pdb.h you can call into mscorpdb.dll.
 
-##What is a PDB
+<h2>What is a PDB</h2><br>
```

Comment:
Same here - to make 2nd level title it should be 
```
## What is a PDB
```

(pay attention to the space between `##` and title)

Reply:
working on it asap